mitodl/open-discussions
# search restyling

#### Pre-Flight checklist

<img width="1680" alt="Screen Shot 2020-06-01 at 10 49 16 AM" src="https://user-images.githubusercontent.com/1934992/83421225-bcea2d00-a3f5-11ea-8bd6-2f464410fe93.png">
<img width="1678" alt="Screen Shot 2020-06-01 at 10 49 24 AM" src="https://user-images.githubusercontent.com/1934992/83421228-be1b5a00-a3f5-11ea-9fcc-b8bc284b435c.png">
<img width="375" alt="Screen Shot 2020-06-01 at 10 49 41 AM" src="https://user-images.githubusercontent.com/1934992/83421229-bf4c8700-a3f5-11ea-8223-bc53199b0d14.png">
<img width="371" alt="Screen Shot 2020-06-01 at 9 42 27 AM" src="https://user-images.githubusercontent.com/1934992/83415933-f159eb00-a3ed-11ea-85f0-9fe57caa7080.png">
<img width="379" alt="Screen Shot 2020-06-01 at 9 42 34 AM" src="https://user-images.githubusercontent.com/1934992/83415936-f159eb00-a3ed-11ea-9e56-40ebaa21a8c8.png">
<img width="1217" alt="Screen Shot 2020-06-01 at 9 43 30 AM" src="https://user-images.githubusercontent.com/1934992/83415939-f28b1800-a3ed-11ea-9c32-c4c69b8e7fc2.png">
<img width="374" alt="Screen Shot 2020-06-01 at 9 43 41 AM" src="https://user-images.githubusercontent.com/1934992/83415942-f323ae80-a3ed-11ea-9ee7-cc7a53c5624f.png">
<img width="1216" alt="Screen Shot 2020-06-01 at 9 44 24 AM" src="https://user-images.githubusercontent.com/1934992/83415943-f3bc4500-a3ed-11ea-84b1-230d6c726553.png">
<img width="378" alt="Screen Shot 2020-06-01 at 9 44 36 AM" src="https://user-images.githubusercontent.com/1934992/83415921-ee5efa80-a3ed-11ea-832a-120df94c53a0.png">

#### What are the relevant tickets?
closes https://github.com/mitodl/open-discussions/issues/2939
closes https://github.com/mitodl/open-discussions/issues/2821
closes https://github.com/mitodl/open-discussions/issues/2822

#### What's this PR do?
This pr 
1.  Replaces the document counts for the audience and availability facets with icons
2. Replaces the price and availability information on learning resource cards with audience and availability icons

#### How should this be manually tested?
Go to /learn/search and verify that the pr looks good and looks like the designs.
Also check that `docker-compose run web ./manage.py recreate_index` still works. There should be no change to the data in the index
